### PR TITLE
Python 3 support

### PR DESCRIPTION
**Pull request by QuLogic**
Best to start this off from the beginning.

You may want to check the classifiers, though.

### CODE REVIEW

**Comment by pelson** [comment]
:+1: - awesome. Thanks @QuLogic!
It would be super awesome to get mo_pack tested with travis-ci on Py 2 + 3. I won't hold this PR up for it, but any volunteers @lbdreyer / @QuLogic ?

File: setup.py [hunk 1]
```diff
@@ -19,4 +21,22 @@
     ext_modules=cythonize(extensions),
     packages=['mo_pack'],
     package_dir={'': 'lib'},
+    classifiers=[
+            'Development Status :: 3 - Alpha',
+            ('License :: OSI Approved :: '
+             'GNU Lesser General Public License v3 or later (LGPLv3+)'),
+            'Operating System :: MacOS :: MacOS X',
+            'Operating System :: Microsoft :: Windows',
```
Comment (pelson) [inline setup.py]: Not tested this one.

Comment (QuLogic) [inline setup.py]: OK, I guess you can add that in when you do.

**Comment by pelson** [comment]
:tada:! Thanks @QuLogic!

**Comment by QuLogic** [comment]
BTW, please update scitools binstar.

**Comment by pelson** [comment]
Tagged a v0.1.0 and https://github.com/SciTools/conda-recipes-scitools/pull/76.

Thanks @QuLogic

**Pull request merged**